CHANGELOG.md

```diff
@@ -1,3 +1,70 @@
+# [0.3.0-beta.1](https://github.com/imgix/imgix-core-js/compare/v0.2.3...v0.3.0-beta.1) (2021-02-03)
```

Comment:
You might think about manually editing this since it's inaccurate

Reply:
Yeah, I've gotta rework this